web-ng/src/app/components/layer-list-item/layer-list-item.component.html

```diff
@@ -19,10 +19,19 @@
   <h3 matLine>
     {{ layer?.name?.get(lang) }}
   </h3>
-  <button mat-icon-button [matMenuTriggerFor]="layerMenu">
-    <mat-icon>more_vert</mat-icon>
-  </button>
-  <mat-menu #layerMenu="matMenu">
-    <button mat-menu-item (click)="onCustomizeLayer()">Customize layer</button>
-  </mat-menu>
+  <div [ngSwitch]="actionsType">
+    <div *ngSwitchCase="layerListItemActionsType.MENU" >
+      <button mat-icon-button [matMenuTriggerFor]="layerMenu">
+        <mat-icon>more_vert</mat-icon>
+      </button>
+      <mat-menu #layerMenu="matMenu">
+        <button mat-menu-item (click)="onCustomizeLayer()">Customize layer</button>
+      </mat-menu>
+    </div>
+    <div *ngSwitchCase="layerListItemActionsType.BACK" >
```

Comment:
Does this conditional appear in the LayerListItemComponent because we reuse it for both the list and the feature details header? If so, can we refactor this component so that we don't reuse it directly in both places? I assume the UI is going to diverge a lot between the list items and the header, so this might make more sense. Wdyt?

Reply:
Please disregard - we'll return to this once we know what the header will look like in other panels.

Reply:
I added TODO to change it when we know UI as we discussed ofline